### PR TITLE
Compute and Network CLI Commands: Features and bugs fixes

### PR DESCRIPTION
**Pull request by ruslangabitov**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ruslangabitov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by yugangw-msft** [comment]
2 initial feedback, 
1. please get rid of "lib/commands/arm/armsdk/networkResourceProviderClient.js" and replace with package references. Ask service team to publish if they have not.
2. please get rid of "lib/commands/arm/armsdk/trafficmanager*

**Comment by azurecla** [comment]
__@ruslangabitov__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by markcowl** [comment]
@ruslangabitov tests are failing, please fix

**Comment by yugangw-msft** [comment]
I still saw the "lib/commands/arm/armsdk/networkResourceProviderClient.js" and "lib/commands/arm/armsdk/trafficmanager". Can we delete them?

File: test/commands/cli.vm.capture-tests.js [hunk 1]
```diff
@@ -23,100 +23,100 @@ var testPrefix = 'cli.vm.capture-tests';
...
-    });
+        before(function(done) {
+            suite = new CLITest(this, testPrefix, requiredEnvironment);
+            suite.setupSuite(function() {
+			    location = process.env.AZURE_VM_TEST_LOCATION;
+                timeout = suite.isPlayback() ? 0 : testUtils.TIMEOUT_INTERVAL;
+                certFile = process.env.SSHCERT;
+                retry = 5;
+				done();
+			});
```
Comment (yugangw-msft) [inline test/commands/cli.vm.capture-tests.js]: fix the indent and across the test files

File: test/data/appgatewayconfig.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
+    "backendAddressPools": [{
```
Comment (yugangw-msft) [inline test/data/appgatewayconfig.json]: json file indent is 2 spaces

**Pull request merged**